app/src/main/java/com/github/libretube/enums/ContentFilter.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.libretube.enums
+
+import com.github.libretube.constants.PreferenceKeys.SELECTED_FEED_FILTERS
+import com.github.libretube.helpers.PreferenceHelper
+
+private val enabledFiltersSet get() = PreferenceHelper
+    .getString(SELECTED_FEED_FILTERS, "1,2,3")
+    .split(',')
+    .toMutableSet()
+
+enum class ContentFilter(private val id: Int) {
```

Comment:
We don't actually need this "id" parameter here.
Instead we can use `ContentFilter#ordinal` which returns the index of the enum property, so:
VIDEOS -> 0
SHORTS -> 1
LIVESTREAMS -> 2